lbrynet/dht/protocol.py

```diff
@@ -253,8 +253,18 @@ def datagramReceived(self, datagram, address):
                     else:
                         exception_type = UnknownRemoteException
                     remoteException = exception_type(message.response)
-                    log.error("Remote exception (%s): %s", address, remoteException)
-                    df.errback(remoteException)
+                    # this error is returned by nodes that can be contacted but have an old
+                    # and broken version of the ping command, if they return it the node can
+                    # be contacted, so we'll treat it as a successful ping
+                    old_ping_error = "ping() got an unexpected keyword argument '_rpcNodeContact'"
```

Comment:
Put in comments here what version of the daemon has the "old and broken version of the ping command"  (16.4 right?) 
I don't think we want to keep this around forever

Reply:
just a thought here: what happens if a rpc method is invoked on a node that does not implement that method at all?

Imagine in 0.21 a new 'XXX' command gets implemented.
Then a 0.21 node tries to call 'XXX' on an older one. What will the latter say? 
Will it send an error message or will the node ignore it gracefully? Will the caller node properly handle the error?

(sorry if this problem was already discussed/solved elsewhere, but it just popped up while reading this chunk)

Reply:
It'll raise an attribute error and return it as an error response (which will re-raise the attribute error for the calling client)